flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/nodes/CygnusApplication.java

```diff
@@ -134,6 +138,12 @@ public static void main(String[] args) {
                 mgmtIfPort = new Integer(commandLine.getOptionValue('p')).intValue();
             } // if
             
+            int pollingTime = 30; // default value
```

Comment:
Although I have seen in the code that this is the way it is done with other values, shouldn't this be a good change to group the "default values" as constants at the beginning of the class? I leave at your discretion ;)